tests/Model/Generated/Object/NotificationUrlTest.php

```diff
@@ -0,0 +1,282 @@
+            MonetaryAccountBank::class,
+            self::GET_MONETARY_ACCOUNT_BANK
+        );
+    }
+
+    /**
+     */
+    public function testPaymentBatchModel()
+    {
+        $this->executeTest(
+            self::JSON_PATH_PAYMENT_BATCH_MODEL,
+            PaymentBatch::class,
+            self::GET_PAYMENT_BATCH
+        );
+    }
+
+    /**
+     */
+    public function testRequestResponse()
+    {
+        $this->executeTest(
+            self::JSON_PATH_REQUEST_RESPONSE_MODEL,
+            RequestResponse::class,
+            self::GET_REQUEST_RESPONSE
+        );
+    }
+
+    /**
+     */
+    public function testRequestInquiryModel()
+    {
+        $this->executeTest(
+            self::JSON_PATH_REQUEST_INQUIRY_MODEL,
+            RequestInquiry::class,
+            self::GET_REQUEST_INQUIRY
+        );
+    }
+
+    /**
+     */
+    public function testSchedulePaymentModel()
+    {
+        $this->executeTest(
+            self::JSON_PATH_SCHEDULE_PAYMENT_MODEL,
+            SchedulePayment::class,
+            self::GET_SCHEDULE_PAYMENT
+        );
+    }
+    
+    /**
+     */
+    public function testShareInviteBankResponse()
+    {
+        $this->executeTest(
+            self::JSON_PATH_SHARE_INVITE_BANK_RESPONSE_MODEL,
+            ShareInviteBankResponse::class,
+            self::GET_SHARE_INVITE_BANK_RESPONSE
+        );
+    }
+
+    /**
+     */
+    public function testScheduledInstance()
+    {
+        $this->executeTest(
+            self::JSON_PATH_SCHEDULE_INSTANCE_MODEL,
+            ScheduleInstance::class,
+            self::GET_SCHEDULE_INSTANCE
+        );
+    }
+
+    /**
+     */
+    public function testShareInviteBankInquiry()
+    {
+        $this->executeTest(
+            self::JSON_PATH_SHARE_INVITE_BANK_INQUIRY_MODEL,
+            ShareInviteBankInquiry::class,
+            self::GET_SHARE_INVITE_BANK_INQUIRY
+        );
+    }
+
+    /**
+     */
+    public function testMasterCardAction()
+    {
+        $this->executeTest(
+            self::JSON_PATH_MASTER_CARD_ACTION_MODEL,
+            MasterCardAction::class,
+            self::GET_MASTER_CARD_ACTION
+        );
+    }
+
+    /**
+     */
+    public function testChatMessageAnnouncementModel()
+    {
+        $this->executeTest(
+            self::JSON_PATH_CHAT_MESSAGE_ANNOUNCEMENT_MODEL,
+            ChatMessageAnnouncement::class,
+            self::GET_CHAT_MESSAGE_ANNOUNCEMENT
+        );
+    }
+
+    /**
+     */
+    public function testDraftPaymentModel()
+    {
+        $this->executeTest(self::JSON_PATH_DRAFT_PAYMENT_MODEL, DraftPayment::class, self::GET_DRAFT_PAYMENT);
+    }
+
+    /**
+     * @param string $jsonString
+     *
+     * @return NotificationUrl
+     */
+    private function getNotificationUrlFromJson(string $jsonString): NotificationUrl
+    {
+        $json = json_decode($jsonString, true);
+        static::assertNotNull($json, self::ASSERT_JSON_DECODE_ERROR);
```

Comment:
Assertions like personal space (unless they stick to other assertions) or get anxious otherwise. Please surround this one with newlines :D